sdk/communication/azure-communication-callingserver/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This is the first version of the restart of Azure Communication Service Calling
 - Call recording.
 - Get, add and remove participants from the call.
 - Recording download apis.
+- Added support for an optional `kind` property for a converter of `CommunicationIdentifierModel`.
```

Comment:
`CommunicationIdentifierModel` is not extracted as a part of the public API, we shouldn't mention it here. 
```suggestion
- Optimized the (de)serialization of the types derived from the `CommunicationIdentifier`.
```